### PR TITLE
Slight module changes + unit testing

### PR DESCRIPTION
**Pull request by twallie**
Added a setup.py script to allow for installation with pip and added a basic conversion test.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .vscode/
 env/
-__pycache__/
\ No newline at end of file
+__pycache__/
+youtube_to_osu.egg-info/
```
Comment (Cgembarow) [inline .gitignore:4]: nice

File: README.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ virtualenv env
 source env/bin/activate
 pip install -r requirements.txt
 ```
+
 ... then test with Python interpreter
+
```
Comment (GuyWhoC0des) [inline README.md:14]: lots of empty space...

File: README.md [hunk 3]
```diff
@@ -18,5 +20,13 @@ Type "help", "copyright", "credits" or "license" for more information.
 Number of notes: 1127
 Song length (seconds): 252.3544671201814
 Average NPS: 4.465940361036989
->>> 
+>>>
+```
+
+## Testing
```
Comment (GuyWhoC0des) [inline README.md:26]: testing what? be specific. captivate me

File: requirements.txt [hunk 4]
```diff
@@ -1,7 +1,7 @@
 aubio==0.4.9
 ffmpeg==1.4
 ffmpeg-python==0.2.0
-future=0.18.3
+future==0.18.3
```
Comment (GuyWhoC0des) [inline requirements.txt:4]: trivial mistakes...

**Comment by twallie** [comment]
Don't delete this branch, or close the pull request. I'm actually still working on the mp3 additions we talked about.

**Comment by twallie** [comment]
@GuyWhoC0des please review my code please please please please please please please please please please please please please please please please

File: LICENSE
Comment (twallie) [inline LICENSE:1]: Do you think it'll work?

**Comment by twallie** [comment]
Also PLEASE for the love of GOD change the repository name

File: .gitignore (hunk 1 above)
Comment (twallie) [reply .gitignore:4]: Thanks for the really direct feedback.

File: LICENSE
Comment (Cgembarow) [inline LICENSE:1]: Thomas, this may lead to legal-troubles. Have u ran this by HR?

**Review by Cgembarow — APPROVED** [review]
Great addition. you are really making a name for yourself at this company

File: LICENSE
Comment (twallie) [reply LICENSE:1]: Has your mom been ran by HR?

**Pull request merged**